editor/editor_settings.cpp

```diff
@@ -619,6 +622,9 @@ bool EditorSettings::_save_text_editor_theme(String p_file) {
 	cf->set_value(theme_section, "search_result_color", ((Color)get("text_editor/highlighting/search_result_color")).to_html());
 	cf->set_value(theme_section, "search_result_border_color", ((Color)get("text_editor/highlighting/search_result_border_color")).to_html());
 
+	//GDScript highlighter
+	cf->set_value(theme_section, "GDScript/function_definition_color", ((Color)get("text_editor/highlighting/GDScript/function_definition_color")).to_html());
```

Comment:
Maybe it should be made possible for SyntaxHighlighters to register such options themselves, so that it can be done also for other languages which are added as GDNative modules.

Reply:
Yeah I've been thinking about this but can't arrive at a decent solution. It would need both adding here and under `editor_themes.cpp` for light / dark theme switching.

Reply:
While this would be a nice addition, I think it's out of scope of this PR. I believe it'll need a bit of a refactor for EditorSettings to allow modules registering settings in a timely manner.